buildSrc/src/main/kotlin/AndroidBuildGradlePlugin.kt

```diff
@@ -14,7 +14,7 @@ class AndroidBuildGradlePlugin : Plugin<Project> {
 
     override fun apply(project: Project) {
         project.android().apply {
-            compileSdk = 34;
+            compileSdk = 35;
             defaultConfig {
                 minSdk = 23
```

Comment:
should we need to support 23 ? i heard the Android 15 lower than 24 can't be installed.

Reply:
I have tested it on Android 15, it works fine for now.